packages/devtools/src/schema/parser.ts

```diff
@@ -326,6 +326,7 @@ export class SchemaParser {
     for (const [key, value] of Object.entries(objectFields)) {
       const directives = getDirectives(this.#schema, value)
 
+      const immutable = directives.some((item) => item.name === 'immutable')
```

Comment:
Does the directive check need to be here, rather than part of the `_parseListType()` method?

Reply:
could be in either place I think I put it here wanted it to look homogeneous to another parse method, but Ill put it directly in the method

Reply:
Done!